libs/couch2pg/importer.js

```diff
@@ -0,0 +1,105 @@
+var _ = require('underscore'),
+    Promise = require('rsvp').Promise,
+    format = require('pg-format');
+
+var deleteDocuments = function(db, docIdsToDelete) {
+  if (docIdsToDelete && docIdsToDelete.length > 0) {
+    return db.query(
+      format('DELETE FROM couchdb WHERE doc->>\'_id\' in (%L)',
+        docIdsToDelete));
+  } else {
+    return Promise.resolve();
+  }
+};
+
+var storeSeq = function(db, seq) {
+  return db.query(format('UPDATE couchdb_progress SET seq = %L', seq));
```

Comment:
I almost wonder if we could add a JSON object to the `couchdb` table with this same information, rather than create a whole new table for a single value. On the other hand, this gives us a place to grow and avoids possible quantum implosion events from storing the seqnum with all the stuff that gets fetched using the seqnum.